src/Exceptions/DatasetAlreadyExist.php

```diff
@@ -17,8 +17,8 @@ final class DatasetAlreadyExist extends InvalidArgumentException implements Exce
     /**
      * Creates a new Exception instance.
      */
-    public function __construct(string $name)
+    public function __construct(string $name, string $scope)
```

Comment:
@nunomaduro shall we use the v2 opportunity to rename this to `DatasetAlreadyExists`? Grammatically correct :-)

Reply:
It's an internal class, so don't think there are any issues in doing it